src/test/java/com/rackspacecloud/metrics/ingestionservice/CreateInfluxDBTests.java

```diff
@@ -0,0 +1,224 @@
+        List<QueryResult.Result> bkdbbresults = influxDB.query(bkquery).getResults();
+        List<List<Object>> bkdbtble = bkdbbresults.get(0).getSeries().get(0).getValues();
+
+        for (int i = 0; i < bkdbtble.size(); i++) {
+            List<Object> backuptble = bkdbtble.get(i);
+            for (Object bktable : backuptble) {
+                QueryResult bkrowcount = influxDB.query(new Query("SELECT count(*) FROM " + bktable, "db_0"));
+                List<QueryResult.Result> bkrowcountResults = bkrowcount.getResults();
+                // System.out.println("rowcounts of table in backup db  db_0  ----" + bkrowcountResults);
+
+                if(bkrowcountResults.get(0).hasError()) {
+                }else{
+                    List<List<Object>> backupdbtblecnt = bkrowcountResults.get(0).getSeries().get(0).getValues();
+                    // Get the row count of each table from the restore database
+
+                    for (int j = 0; j < backupdbtblecnt.size(); j++) {
+                        List<Object> backuprowcount = backupdbtblecnt.get(j);
+                        bkuprowcountval = Float.valueOf(backuprowcount.get(1).toString());
+                        bkrwcnt.add(Float.valueOf(backuprowcount.get(1).toString()));
+                    }
+
+                }
+
+            }
+        }
+
+        Query query = new Query("SHOW MEASUREMENTS", "db_1");
+        List<QueryResult.Result> restoredbresults = influxDB.query(query).getResults();
+        List<List<Object>> restoredbtble = restoredbresults.get(0).getSeries().get(0).getValues();
+
+        for (int i = 0; i < restoredbtble.size(); i++) {
+            List<Object> restoretble = restoredbtble.get(i);
+            for (Object restoretable : restoretble) {
+                //QueryResult rowcount = influxDB.query(new Query("SELECT count(*) FROM /.*/" , "db_1"));
+                QueryResult rowcount = influxDB.query(new Query("SELECT count(*) FROM "+ restoretable  , "db_1"));
+                List<QueryResult.Result> restoretbleresults =rowcount.getResults();
+                if(restoretbleresults.get(0).hasError()) {
+                }else{
+                    List<List<Object>> resdbtblecnt = restoretbleresults.get(0).getSeries().get(0).getValues();
+                    //  System.out.println("RowCount in restore db table  "  + resdbtblecnt);
+
+                    // Get the row count of each table from the restore database
+
+                    for (int j = 0; j < resdbtblecnt.size(); j++) {
+                        List<Object> resrowcount = resdbtblecnt.get(j);
+                        //   System.out.println("Row count from each table in restore db " + resrowcount.get(1));
+                        resrowcountval = Float.valueOf(resrowcount.get(1).toString());
+                        resrwcnt.add(Float.valueOf(resrowcount.get(1).toString()));
+                        //System.out.println("Row count from each table in restore db " + resrowcountval);
```

Comment:
@hravi76 code from line 187-201  is repeating code on lines 152-177. Please create a helper method to compare row count.